docgen/assets/js/bindRunExamples.js

```diff
@@ -2,16 +2,21 @@ import instantsearch from "../../../src/lib/main.js";
 import algoliasearch from "algoliasearch/lite";
 import capitalize from "lodash/capitalize";
 
+const {
+  appId = "latency",
+  apiKey = "6be0576ff61c053d5f9a3225e2a90f76",
+  ...searchConfig
+} =
+  window.searchConfig || {};
```

Comment:
Weird indentation, it should be on the same line no?

Reply:
Prettier does that... I'm not sure why either.

Reply:
Weird I tried in the REPL and I didn't have this.